test/onigumo_cli_test.exs

```diff
@@ -23,8 +28,10 @@ defmodule OnigumoCLITest do
       assert usage_message_printed?(fn -> Onigumo.CLI.main(["Downloader", "Parser"]) end)
     end
 
-    test("run CLI with invalid switch") do
-      assert usage_message_printed?(fn -> Onigumo.CLI.main(["--invalid"]) end)
+    for switch <- @invalid_switches do
+      test("run CLI with invalid switch #{switch}") do
+        assert usage_message_printed?(fn -> Onigumo.CLI.main([unquote(switch)]) end)
+      end
```

Comment:
We don’t use exit codes! 😱  It struck me when I saw them not being tested. Not for this PR, but created an issue #224.